backend/blueprint/api/controllers/registerUser.py

```diff
@@ -29,14 +27,6 @@ def registerUser():
         email=user.email,
```

Comment:
why `email` is mentioned twice in `auth.create_user`?

Reply:
Sorry, but what ?

Reply:
There is a user for backend and a user for frontend. 
I guess you should look more closely :confused:

Reply:
@gabru-md ignore this comment, I have seen something wrong

Reply:
One of many